src/components/panel/demoGroups/index.html

```diff
@@ -0,0 +1,44 @@
+<div ng-controller="PanelGroupsCtrl as ctrl" ng-cloak>
+
+  <md-toolbar class="md-accent">
+    <div class="md-toolbar-tools">
+      <md-button
+        class="md-icon-button"
+        aria-label="Settings"
+        ng-click="ctrl.showMenu($event, 'settings', 'toolbar')">
```

Comment:
@ErinCoughlan This was originally going to be a larger demo where another group was going to be in the content area below the toolbar. The `showMenu` function was opened up in order to allow for multiple groups to be added and their panels opened. I do not understand your second paragraph.